features/apimgt/org.wso2.carbon.apimgt.publisher.feature/src/main/resources/publisher/source/src/app/components/Apis/Listing/components/APICreateMenu.jsx

```diff
@@ -18,128 +18,291 @@
+                                </Box>
+                                {(deploying !== null && handleDeploySample !== null) && (
+                                    <Box mt={2}>
+                                        {!deploying ? (
+                                            <Typography variant='body1'>
+
+                                                <MUILink
+                                                    id='itest-id-createdefault'
+                                                    onClick={handleDeploySample}
+                                                    className={classes.links}
+                                                >
+                                                    <FormattedMessage
+                                                        id={'Apis.Listing.SampleAPI.SampleAPI.'
+                                                + 'rest.d.sample.title'}
+                                                        defaultMessage='Deploy Sample API'
+                                                    />
+                                                </MUILink>
+                                            </Typography>
+                                        )
+                                            : (
+                                                <CircularProgress
+                                                    size={24}
+                                                    className={classes.buttonProgress}
+                                                />
+                                            )}
+                                        <Typography variant='body2'>
+                                            <FormattedMessage
+                                                id='Apis.Listing.SampleAPI.SampleAPI.rest.d.sample.content'
+                                                defaultMessage={`This is a sample API for Pizza Shack 
+                                    online pizza delivery store`}
+                                            />
+                                        </Typography>
+                                    </Box>
+                                )}
+
+
+                            </Box>
+
+                        </Grid>
+                        <Grid item xs={12} sm={6} md={3} lg={3}>
+                            <Box textAlign='center' mt={2}>
+                                <Typography variant='subtitle' component='div'>
+                                    <FormattedMessage
+                                        id='Apis.Listing.SampleAPI.SampleAPI.soap.api'
+                                        defaultMessage='SOAP API'
+                                    />
+                                </Typography>
+                                <img
+                                    src={Configurations.app.context
+                                + '/site/public/images/landing-icons/soapapi.svg'}
```

Comment:
Shall we move this svg location or the icon name to the theme file? so in case someone needs to change it , they can do it by changing the filename or path

Reply:
done